dockerfiles/che/entrypoint.sh

```diff
@@ -324,14 +323,60 @@ add_che_cert_to_truststore() {
 }
 
 add_public_cert_to_truststore() {
+  JAVA_TRUST_STORE=/home/user/cacerts
+  DEFAULT_JAVA_TRUST_STOREPASS="changeit"
+
+  chmod 644 $JAVA_TRUST_STORE
```

Comment:
why are we setting this to `644` AGAIN, and what's the fallback when on openshift this fails due to the anyuid user not having permission to `chmod` this file? (See above, https://github.com/eclipse/che/pull/18504/files#diff-0494d7f08e22baf0f7a6a136eb6f5a6a6532d3199d4edda56964dae41c18b22bL309

Reply:
We set `644` to be sure it is allowed to change keystore content below.
There is no fallback as it will not fait due to  anyuid user: https://github.com/eclipse/che/pull/18504/files#diff-0494d7f08e22baf0f7a6a136eb6f5a6a6532d3199d4edda56964dae41c18b22bR301